docs/modules/integrations/pages/github-action/run-appium-script.adoc

```diff
@@ -0,0 +1,18 @@
+= Run Appium script on Kobiton devices
+:navtitle: Run Appium script
+
+Learn how to utilize Kobiton automation testing with Appium script and GitHub Action.
+
+NOTE: For more information, see link:https://github.com/marketplace/actions/kobiton-execute-appium-test[Official GitHub Marketplace Kobiton Action for Appium].
+
+== Before you start
+
+You'll need the following:
+
+* A Kobiton account. To create a Kobiton account visit link:https://portal.kobiton.com/register[Kobiton].
+* Kobiton Username and API key. To get your Kobiton Username and API Key, follow instructions at `IV. Configure Test Script` section on link:https://kobiton.com/blog/parallel-testing-with-selenium-webdriver/[Kobiton blog].

```

Comment:
Try to link to pages within the same site. In this case it should be linked to `profile/manage-your-api-credentials#_get_an_api_key`

Reply:
Linked within the same site